README.md

```diff
@@ -41,7 +41,7 @@ It is currently in active development in the Kubernetes community through the [d
 | ---------------------------------------------------- | --------------------------------------------------------------------------|-----|
 | [crio(8)](/docs/crio.8.md)                           | OCI Kubernetes Container Runtime daemon                                   ||
 | [kpod(1)](/docs/kpod.1.md)                           | Simple management tool for pods and images                                ||
-| [kpod-cp(1)](/docs/kpod-cp.1.md)                     | Copy files/folders between a container and the local filesystem           ||
+| [kpod-cp(1)](/docs/kpod-cp.1.md)                     | kpod doesn't implement the `kpod cp` command, this man page descibes how to use `kpod mount` to get the same behaviour.||
```

Comment:
Perhaps:
Instead of providing a cp command, the `kp mount` command page describes how to use 'kpod mount' to have even more flexibility and functionality.

I really don't like getting hit with "doesn't" right out of the blocks.  Too negative in my book.  That said, feel free to change up my suggestion, I'm not totally happy with that either, but need to run atm.